### PR TITLE
[CBRD-24563] Add new test case for regular expression library change.

### PR DESCRIPTION
**Pull request by tw-kang**
http://jira.cubrid.org/browse/CBRD-24563

Verified the answer modification with regex101.com
Used Golang as FLAVOR to verify RE2 library.

### CODE REVIEW

File: sql/_34_fig/cbrd_24563/cases/REGEXP_COUNT_unicode.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+-- CBRD-24563 : default regexp library change from cppstd to RE2
+set system parameters 'regexp_engine=re2';
+
+--REGEXP_COUNT
+SET NAMES utf8 COLLATE utf8_ko_cs;
+SELECT REGEXP_COUNT('가나다가나다가나다라', '(가나)다', 1);
+SELECT REGEXP_COUNT('가나 가나다라 마바사아 자차카타 파하', '[[:alpha:]]+', 1);
+SELECT REGEXP_COUNT('11억 8112만 5400 won 더하기 420만 4432 won', '[[:alpha:]]', 8);
+SELECT REGEXP_COUNT('Kłak Aleksander', '[[:alpha:]]+', 1);
```
Comment (hgryoo) [inline sql/_34_fig/cbrd_24563/cases/REGEXP_COUNT_unicode.sql:9]: To test such a case that the character class doesn't match the Unicode character, The case is meaningful. Additionally, We should add test cases that have \u{hangul} pattern matching with the Unicode Korean characters.

Please refer to the Google RE2 syntax link that I wrote in the Acceptance Criteria section in JIRA.

File: sql/_34_fig/cbrd_24563/cases/cbrd_24563.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+select ('aabbb' rlike 'ab{0,49995}$');
+select ('aabbb' rlike 'ab{-1,3}$');
+
+-- verify value : default (re2)
+set system parameters 'regexp_engine=default';
+select ('aabbb' rlike 'ab{0,49995}$');
+select ('aabbb' rlike 'ab{-1,3}$');
+
+-- negative case
+set system parameters 'regexp_engine=' '';
```
Comment (tw-kang) [inline sql/_34_fig/cbrd_24563/cases/cbrd_24563.sql:23]: @hgryoo 
if the value is empty should occur an error.
But now it works the same as the former set value.

File: sql/_34_fig/cbrd_24563/cases/REGEXP_COUNT_unicode.sql (hunk 1 above)
Comment (tw-kang) [reply sql/_34_fig/cbrd_24563/cases/REGEXP_COUNT_unicode.sql:9]: In RE2 manual Unicode pattern matching use '\p{Hangul}'. 
The suggested test case is added.

File: sql/_34_fig/cbrd_24563/cases/cbrd_24563.sql (hunk 2 above)
Comment (hgryoo) [reply sql/_34_fig/cbrd_24563/cases/cbrd_24563.sql:23]: The regexp_engine is a PRM_KEYWORD type parameter. If an invalid value is specified, CUBRID ignores it.
Unfortunately, there seems to be no description of the keyword type parameters... :-(

The following parameters are also keyword types like regexp_engine:
```
error_log_level
isolation_level
page_validation_level
ha_mode_for_sa_utils_only
ha_server_state
ha_log_applier_state
compat_mode
check_peer_alive
query_trace_format
fault_injection_test
ha_repl_filter_type
tde_default_algorithm
```

Comment (tw-kang) [reply sql/_34_fig/cbrd_24563/cases/cbrd_24563.sql:23]: The description is added in the test case.

**Review by hgryoo — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

**Review by kwonhoil — APPROVED** [review]
